src/Chainweb/Pact/Types.hs

```diff
@@ -354,7 +356,10 @@ data PactServiceEnv tbl = PactServiceEnv
     , _psBlockHeaderDb :: !BlockHeaderDb
     , _psGasModel :: TxContext -> GasModel
     , _psMinerRewards :: !MinerRewards
+    , _psLocalRewindDepthLimit :: {-# UNPACK #-} !Word64
+    -- ^ The limit of rewind's depth in the `execLocal` command.
```

Comment:
There's no need to `UNPACK` this or any other word-sized field anymore, `-funbox-small-strict-fields` is the default.

Reply:
Good point 👍 , added because of `_psReorgLimit`, will remove for both of them.